### PR TITLE
[Bug]: Fix Undefined constant "GLOB_BRACE" on Alpine 

### LINKED ISSUE #17204: [Bug]: Undefined constant "GLOB_BRACE" on Alpine (musl)
### Pimcore version

11

### Issue

Try installing Pimcore on alpine Linux. Constant `\GLOB_BRACE` is not supported there and therefore does not exist. So the installer fails. Regression of https://github.com/pimcore/pimcore/pull/16798

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
fixes https://github.com/pimcore/pimcore/issues/17204

### CODE REVIEW

File: bundles/InstallBundle/src/Installer.php [hunk 1]
```diff
@@ -730,7 +730,7 @@ public function setupDatabase(Connection $db, array $userCredentials, array $err
 
     protected function getDataFiles(): array
     {
-        return glob(PIMCORE_PROJECT_ROOT . '/dump/*{.sql,.sql.gz}', \GLOB_BRACE);
+        return glob(PIMCORE_PROJECT_ROOT . '/dump/*{.sql,.sql.gz}', (\defined('GLOB_BRACE') ? \GLOB_BRACE : 0));
```
Comment (jdreesen) [inline bundles/InstallBundle/src/Installer.php]: But does it find anything if we don't set the flag?

Comment (jdreesen) [reply bundles/InstallBundle/src/Installer.php]: If not, a solution would be:

```suggestion
        if (\defined('GLOB_BRACE')) {
            return glob(PIMCORE_PROJECT_ROOT . '/dump/*{.sql,.sql.gz}', \GLOB_BRACE);
        }
        
        return array_merge(
            glob(PIMCORE_PROJECT_ROOT . '/dump/*.sql'),
            glob(PIMCORE_PROJECT_ROOT . '/dump/*.sql.gz'),
        );
```

Comment (dpfaffenbauer) [reply bundles/InstallBundle/src/Installer.php]: I got that solution from Symfony Finder, so maybe we could levarage finder?

**Comment by blankse** [comment]
@dpfaffenbauer The target branch should now be changed to `11.3`.

**Comment by dpfaffenbauer** [comment]
base changed

File: bundles/InstallBundle/src/Installer.php (hunk 1 above)
Comment (kingjia90) [reply bundles/InstallBundle/src/Installer.php]: Jacob has a point, if it is searching by `{.sql,.sql.gz}` pattern, the `GLOB_BRACE` is mandatory to properly work.
Tested locally and returns nothing with the fallback to `0`. 

Maybe 2 glob calls is not that ideal, what about just
```suggestion
        return glob(PIMCORE_PROJECT_ROOT . '/dump/*.sql*');
```
?

Comment (dpfaffenbauer) [reply bundles/InstallBundle/src/Installer.php]: is it working with that? than its fine.

**Pull request merged**